### DISCUSSION TITLE
Turbo error encountered in Webstorm (only) when using pre-commit hook: Package tasks (<package>#<task>) are not allowed in single-package repositories

### DISCUSSION THREAD

**Discussion by jcollum-nutrien** [vercel/turborepo#5594] [Help] [answered]
https://youtrack.jetbrains.com/issue/WEB-62117/Turbo-error-encountered-in-Webstorm-only-when-using-pre-commit-hook-Package-tasks-packagetask-are-not-allowed-in-single-package?clearDraft=true&description=%0A%0AWS-231.9225.18,%20JRE%2017.0.7%2B10-b829.16x64%20JetBrains%20s.r.o.,%20OS%20Mac%20OS%20X(aarch64)%20v13.4.1,%20screens%202560.0x1440.0,%202560.0x1440.0

Turbo 1.10.10 

Full error: 

```ERROR  run failed: Package tasks (<package>#<task>) are not allowed in single-package repositories: found graphql-schema#build
Turbo error: Package tasks (<package>#<task>) are not allowed in single-package repositories: found graphql-schema#build
husky - pre-commit hook exited with code 1 (error)
```
This does not happen when I use `git commit` with Husky hooks via the command line. 

I'm trying to sort out if this is a Turbo issue or a Webstorm issue and really trying to head off any "well it's the other person's bug" answers. This only happens in WS so I think it's their bug but any insight here would be helpful.

**Comment by anthonyshew** [comment, marked as answer]
Looks like you stopped seeing the issue at that link!
